go.mod

```diff
@@ -11,8 +11,8 @@ require (
 	go.uber.org/automaxprocs v1.6.0
 	google.golang.org/grpc v1.69.0
 	google.golang.org/protobuf v1.36.5
-	k8s.io/api v0.33.1
-	k8s.io/client-go v0.33.1
+	k8s.io/api v0.34.0-alpha.1
+	k8s.io/client-go v0.34.0-alpha.1
```

Comment:
I would probably wait for GA of 1.34. Otherwise dependabot gets crazy and too eager to update to 1.35 alpha, once it's released.